database/migrations/2021_12_09_082449_create_orders_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateOrdersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('orders', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('customer_id');
+            $table->foreign('customer_id')->references('id')->on('customers')->restrictOnDelete()->cascadeOnUpdate();
+            $table->unsignedInteger('payment_id')->unique();
+            $table->foreign('payment_id')->references('id')->on('payments')->restrictOnDelete()->cascadeOnUpdate();
+            $table->unsignedDecimal('totat_price', 10, 2);
+            $table->timestamp('order_date');
```

Comment:
why not datetime?

Reply:
In this case i thought that timestamp is datetime itself.